ui/cypress/integration/node/nodelist.spec.js

```diff
@@ -0,0 +1,34 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Node list', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('api/kubernetes/api/v1/nodes', 'fixture:kubernetes/nodes.json');
```

Comment:
If this route is expected to be useful _in general_, then I'd suggest you define it in `cy.setupMocks`